### PR TITLE
setup project skeleton

### PR DESCRIPTION
**Pull request by Oghenebrume50**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi, I used environment variables to hide my keys, so I am guessing the reviewer would have registered on these platforms before now, anyways the image API was gotten from pixabay, the weather was as stated by the odin project

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const weatherCall = () => {
+  const location = document.getElementById('location').value;
+  const unit = document.getElementById('unit').value;
+  const weatherData = `http://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=${weatherAPI}`;
+  fetch(weatherData, {
+    mode: 'cors',
+  }).then((data) => data.json()).then((res) => {
+    switch (res.cod) {
+      case 200: fetchImage(res.weather[0].main);
```
Comment (SunnyAsar) [inline src/index.js]: All this is too coupled so, lets decouple:
- lets have a method that is responsible just  getting the data
- another to prepare and put the data in the UI

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        display.innerHTML = '';
+        display.innerHTML += `
+      <div class="infoHold">
+      <span id="1">City: ${res.name}</span>
```
Comment (SunnyAsar) [inline src/index.js]: lets use de structuring. then prepare our data

File: src/displayWeather.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+const display = document.getElementById('display');
+
+const displayWeather = async () => {
+  const { cod, main, weather, unit, name } = await fetchData();
+  switch (cod) {
+    case 200: display.innerHTML = '';
+      display.innerHTML += `
+    <div class="infoHold">
+    <span id="1">City: ${name}</span>
```
Comment (SunnyAsar) [inline src/displayWeather.js]: won't it be nicer if we had this UI updated in a dedicated method?
[maybe/Suggestion]: status good  call a ui method and pass the data to it.
else call another method and tell it the problem.

**Note:**
implement as you like but separate concerns and let the error tell me whats wrong.

File: src/displayWeather.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      Temperature: ${degreeUnit(unit, main.temp)}
+    </span>
+    <span id="1">Description: ${weather.description}</span>
+    <span id="2">Outcome: ${weather.main}</span>
+    </div>
+  `;
+      display.innerHTML += await fetchImage(weather.main);
+      break;
+    default: display.innerHTML = '';
+      display.innerHTML += error();
```
Comment (SunnyAsar) [inline src/displayWeather.js]: this error is not useful. its a 404 image but that should be letting you know the city name is wrong or something like that.

**Pull request merged**